CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
```

Comment:
FYI, I think you might have missed the point of the brackets. :stuck_out_tongue_winking_eye: Each version links to the complete revision history for that version on GitHub: https://github.com/olivierlacan/keep-a-changelog/blob/master/CHANGELOG.md

Look at the end of the file here for an example of how that's done in the Markdown source: https://raw.githubusercontent.com/olivierlacan/keep-a-changelog/master/CHANGELOG.md